public/app/percona/backup/components/ScheduledBackups/ScheduledBackups.tsx

```diff
@@ -148,6 +166,7 @@ export const ScheduledBackups: FC = () => {
           cronExpression,
           backupName,
           description,
+          retention!,
```

Comment:
What does the exclamation mark mean here?

Reply:
Retention is an optional field, because it's present on the modal for scheduled backups, but not on regular backups.
But because we are taking care of scheduled backups here, we're sure it won't be undefined, so we're telling TS that by using "!".

Reply:
(another solution would be to just do `retention: retention || 0`

Reply:
Had no idea TS allowed `!` in front of variables to do this. Thanks!!